### PR TITLE
3.14.0

### PR DESCRIPTION
**Pull request by allieorth**
Added icons:
`webhook`
`miscellaneous-collection`
`thumbs-up`

Screen Shot 2020-11-12 at 11 40 20 AMScreen Shot 2020-11-12 at 11 40 52 AMScreen Shot 2020-11-12 at 11 41 12 AM

### CODE REVIEW

File: docs/keywords.json [hunk 1]
```diff
@@ -7014,224 +7014,267 @@
...
+      "event",
+      "information",
+      "manage",
+      "create",
+      "http request",
+      "payload",
+      "application",
+      "push"
+    ],
+    "category":"04_Symbols_B",
```
Comment (paulcpederson) [inline docs/keywords.json:7250]: Are these real category names?

**Review by paulcpederson — APPROVED** [review]

File: docs/keywords.json (hunk 1 above)
Comment (allieorth) [reply docs/keywords.json:7250]: Those are how we have our files labeled — but you're completely right that we usually just have the base name without numbers or letters so ill make that change. Thanks

**Review by KatelynSeitz — APPROVED** [review]
Changes look good! ✨

**Review by johnmgriffith — APPROVED** [review]
Looks great! 🚢

**Pull request merged**